lib/bios/seabios.robot

```diff
@@ -0,0 +1,305 @@
+
+Get Hidden Flash Lockdown Submenu Construction
+    [Arguments]    ${checkpoint}=Esc=Exit    ${lines_top}=1    ${lines_bot}=1    ${opt_only}="${FALSE}"
+    # In most cases, we need to strip two lines:
+    #    TOP:
+    #    Title line, such as:    Dasharo System Features
+    #    BOTTOM:
+    #    Help line, such as:    F9=Reset to Defaults    Esc=Exit
+    ${submenu}=    Get Menu Construction    ${checkpoint}    ${lines_top}    ${lines_bot}
+    # Handling of additional exceptions appearing in submenus:
+    #    1. Drop unselectable strings from Device Manager
+    Remove Values From List    ${submenu}    Devices List
+
+    IF    ${opt_only} == ${TRUE}
+        # Handling exceptions caused by some options splitting into multiple lines.
+        # For Dasharo System Features options, we can assume that each entry has
+        # either ">", or "[ ]", or "< >". For other edk2 menus, this is not always
+        # the case (yet?).
+        FOR    ${entry}    IN    @{submenu}
+            ${status}=    Check If Menu Line Is An Option    ${entry}
+            IF    ${status} != ${TRUE}
+                Remove Values From List    ${submenu}    ${entry}
+            END
+        END
+    END
+    RETURN    ${submenu}
+
+Enter Menu From Snapshot
+    [Documentation]    Enter given Menu option
+    [Arguments]    ${menu}    ${option}
+    ${key}=    Extract Menu Key    ${menu}    ${option}
+    Write Bare Into Terminal    ${key}
+
+Enter Menu From Snapshot And Return Sortbootorder Construction
+    [Documentation]    Enter given sortbootorder Menu option and return construction
+    [Arguments]    ${menu}    ${option}
+    ${key}=    Extract Sortbootorder Menu Key    ${menu}    ${option}
+    Write Bare Into Terminal    ${key}
+    ${menu}=    Get Sortbootorder Menu Construction
+    RETURN    ${menu}
+
+Extract Menu Key
+    [Documentation]    Extract key which should be hit to enter given Menu in SeaBIOS
+    [Arguments]    ${menu}    ${option}
+    FOR    ${item}    IN    @{menu}
+        ${matches}=    Run Keyword And Return Status
+        ...    Should Match    ${item}    *${option}*
+        IF    ${matches}
+            ${option}=    Set Variable    ${item}
+            BREAK
+        END
+    END
+    ${key}=    Set Variable    ${option.split('.')[0]}
+    RETURN    ${key}
+
+Extract Sortbootorder Menu Key
+    [Documentation]    Extract key which should be hit to toggle given sortbootorder Menu option
+    [Arguments]    ${menu}    ${option}
+    FOR    ${item}    IN    @{menu}
+        ${matches}=    Run Keyword And Return Status
+        ...    Should Match    ${item}    *${option}*
+        IF    ${matches}
+            ${option}=    Set Variable    ${item}
+            BREAK
+        END
+    END
+    ${key}=    Set Variable    ${option.split()[0]}
+    RETURN    ${key}
+
+Save Sortbootorder Changes
+    [Documentation]    This keyword saves introduced changes
+    Write Bare Into Terminal    s
+
+Get IPXE Boot Menu Construction
```

Comment:
The same as UEFI

Reply:
@macpijan there are too many leftovers I have clean that up: a9ac47cb55f7

Reply:
Ok. Please ping me once you believe you have a complete list of keywords you need, or at least remove those that are not needed at the time, so we do not waste time discussing something to be dropped anyway.

Reply:
@pietrushnic I saw a new MR, do you consider this one finalized per your current needs?

Reply:
@macpijan Yes, I need to figure out how to extend it without starting to enable new tests. If you think something is missing here, considering there is minimal support and new PRs coming, please let me know, and I will try to improve.

Feedback on whether the approach is clean enough would also be helpful.